docs/modules/sql/pages/querying-streams.adoc

```diff
@@ -1,41 +1,60 @@
 = Stream Processing in SQL
-:description: You can run SQL queries on streaming data, using standard SQL as well as some custom functions for features such as streaming aggregations.
+:description: You can run SQL queries on single streams, join a stream with a table, or join two or more streams.
 
-Stream processing is a programming paradigm for continuously performing computations over events as they arrive. The streams are never-ending, which is why they are often described as _unbounded_.
+Stream processing is a programming paradigm for continuously performing computations on data events as they arrive. The streams are never-ending, which is why they are often described as _unbounded_.
 
 {description}
 
 == Context
 
-This guide explains how to query streams, using the following `trades` data as an example:
+This guide explains how to query streams, using the following e-commerce data in examples:
 
 [cols="1m,1m"]
+.Orders
 |===
 |Field|Data type
 
-|trade_amount
+|order_amount
 |BIGINT
 
-|trade_ticker
+|order_id
 |VARCHAR
 
-|price
+|order_time
+|TIMESTAMP
+
+|product_id
+|BIGINT
+
+|unit_price
 |DECIMAL
+|===
 
-|trade_time
+[cols="1m,1m"]
+.Deliveries
+|===
+|Field|Data type
+
+|address
+|VARCHAR
+
+|delivery_id
+|BIGINT
+
+|delivery_time
 |TIMESTAMP
 |===
 
 == Creating a Mapping to a Streaming Source
 
-To allow Hazelcast to read data from a streaming source, you need to create a mapping to it. For example, the mapping for the `trades` Kafka topic may look like the following:
+To allow Hazelcast to read data from a streaming source, you need to create a mapping to it. For example, the mapping for the `orders` Kafka topic might look like this:
 
 ```sql
-CREATE OR REPLACE MAPPING trades (
-  trade_amount BIGINT,
-  trade_ticker VARCHAR,
-  price DECIMAL,
-  trade_time TIMESTAMP)
+CREATE OR REPLACE MAPPING orders (
+  order_amount BIGINT,
+  order_id VARCHAR,
```

Comment:
As a rule, `id` field is some integer type.
```suggestion
  order_id BIGINT,
```

Reply:
Updated as requested.